app/controllers/webhook_controller.rb

```diff
@@ -21,18 +20,13 @@ def callback
     events = client.parse_events_from(body)
     events.each { |event|
       case event
-      when Line::Bot::Event::Message
-        case event.type
-        when Line::Bot::Event::MessageType::Text
-          message = {
-            type: 'text',
-            text: event.message['text']
-          }
-          client.reply_message(event['replyToken'], message)
-        when Line::Bot::Event::MessageType::Image, Line::Bot::Event::MessageType::Video
-          response = client.get_message_content(event.message['id'])
-          tf = Tempfile.open("content")
-          tf.write(response.body)
+      when Line::Bot::Event::Follow
+        user_id = event['source']['userId']
+        if Customer.where(user_id: user_id).empty? and user_id.present?
```

Comment:
[imo]
where文を使うと全レコードに対して捜査することになるので、EXISTS文を使った方がレコード数が増えたときにパフォーマンスがよくなります👀（EXISTS文は見つかったらそれ以上処理が進まないため）

あと、rubyでは一般的に`and`よりも`&&`を使います！（若干挙動も異なります）
https://qiita.com/riku-shiru/items/533a01bdf18e2e3eef46

```suggestion
        if Customer.exists?(user_id: user_id) && user_id.present?
```

Reply:
[FYI] 

find_or_create_by! を使う方法もありそう

```ruby
Customer.find_or_create_by!(user_id: user_id) do |customer|
    customer.name = LineApi.get_user_name(user_id)
end
```